internal/cli/api/api.go

```diff
@@ -329,6 +332,22 @@ func defaultAPIVersion(command api.Command) (string, error) {
 	return lastVersion.Version, nil
 }
 
+func remindUserToPinVersion(cmd *cobra.Command) {
+	versionFlag := cmd.Flag(flag.Version)
```

Comment:
```suggestion
	versionFlag := cmd.Flags().Changed(flag.Version)
```

Reply:
Thanks for the suggestion, but you missed that I also use the flag to get the value a few lines below `versionFlag.Value.String()`